### ISSUE TITLE
Create a "symbols" implementation, similar to what Webflow has

### ISSUE THREAD

**Issue by pajtai** [elementor/elementor#19856]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I'd like to be able to create Elementor section template with overrideable content. Similar functionality can be seen in Webflow's symbols: https://university.webflow.com/lesson/symbols.

An example would be that I make a section template with a CTA that says, "Click here." I create 5 of these on 5 different pages and on the fifth one I want it to linked to a different URL. Then I want to update the background color for the section template, and I want to be able to do so by editing the section template so that it updates all 5 instances.

### Describe the solution you'd like

I'd like section temlpates that can have content that can be overridden without breaking the link to the original section template.

### Describe alternatives you've considered

I've looked at creat custom Elementor widgets using UA: https://unlimited-elements.com/how-to-create-a-custom-elementor-widget/, but this solution uses Twig templates, which doesn't allow the use of standard Elementor widgets.

**Comment by rami-elementor** [comment]
Hi @pajtai ,

Have you tried "**Elementor Global Widget**"?

https://elementor.com/help/global-widget-pro/

**Comment by black-eye** [comment]
+1 Seems similar to this one: https://github.com/elementor/elementor/issues/5252

**Comment by pajtai** [comment]
@rami-elementor yes, but I don't see how you can override content in the global widget. For example, I have a global widget with a CTA link, and I want to change the link on one page. Then I want to change the background color of the global widget for all instances, including the instance with overridden content. As far as I can tell, this can't be accomplished with global widgets. Can it?

**Comment by pajtai** [comment]
@black-eye yes - #5252 is very similar in concept, as is the Webflow implementation linked in the OP.

**Comment by fabianenzensperger** [comment]
> Hi @pajtai ,
> 
> Have you tried using "**Elementor Global Widget**"?
> 
> https://elementor.com/help/global-widget-pro/



> Hi @pajtai ,
> 
> Have you tried using "**Elementor Global Widget**"?
> 
> https://elementor.com/help/global-widget-pro/

@rami-elementor I think @pajtai means something like this: #965

You have announced on YT and FB Presets for all parts of the website in 2021. So this is why we ask about it. And this function would sooooo much useful. I know about Global Widgets, but this ist not almost so good as presets.

**Comment by tekonemedia** [comment]
+1 We need this terribly bad. What's the point of global widgets of they can't have seperate content without breaking the global design link? This is one of the main reasons, (along with containers and loops) that we started to develop with webflow.
